include/ddc/chunk.hpp

```diff
@@ -7,12 +7,16 @@
 #include "ddc/chunk_common.hpp"
 #include "ddc/chunk_span.hpp"
 
+/** A chunck
```

Comment:
```suggestion
/** A chunk
```

Reply:
That was not intended to be here at all, I even think another PR removes this comment